azure/storage/blob/baseblobservice.py

```diff
@@ -240,6 +244,9 @@ def make_blob_url(self, container_name, blob_name, protocol=None, sas_token=None
             blob_name,
         )
 
+        if snapshot:
+            url += '?snapshot=' + snapshot
```

Comment:
thanks @troydai 's review.
if both present, what should the url be?
like this?
`/my.vhd?sas&snapshot=xxxx`
I'm not sure they can be both given.